manifests/m/Microsoft/Teams/Preview/1.5.00.1870/Microsoft.Teams.Preview.installer.yaml

```
@@ -0,0 +1,25 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.Teams.Preview
+PackageVersion: 1.5.00.1870
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: user
+InstallerSwitches:
+  Silent: /s
+  SilentWithProgress: /s
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://statics.teams.cdn.office.net/production-windows-x64/1.5.00.1870/Teams_windows_x64.exe
+  InstallerSha256: 0345C170315563D19FF2C4CC707C72CD8051C744E7F0FAFFC8EF10E831D1E94B
+- Architecture: x86
+  InstallerUrl: https://statics.teams.cdn.office.net/production-windows-x64/1.5.00.1870/Teams_windows_x64.msi
+  InstallerSha256: 2F92A3871C0E72908F80D63CA3AA8EF9A61A1B13C2A08CF0268C55E6B51B39EF
+  ProductCode: '{731F6BAA-A986-45A4-8936-7C3AAAAA760B}'
+- Architecture: arm64
+  InstallerUrl: https://statics.teams.cdn.office.net/production-windows/1.5.00.1870/Teams_windows.exe
+  InstallerSha256: 86D7C89037F741C3AC0F2FA3F1C1DB0BA2E3B910315063625F3F49737011F3CF
```

The msi got removed in #39073
```suggestion
- Architecture: x64
  InstallerUrl: https://statics.teams.cdn.office.net/production-windows-x64/1.5.00.1870/Teams_windows_x64.exe
  InstallerSha256: 0345C170315563D19FF2C4CC707C72CD8051C744E7F0FAFFC8EF10E831D1E94B
- Architecture: x86
  InstallerUrl: https://statics.teams.cdn.office.net/production-windows/1.5.00.1870/Teams_windows.exe
  InstallerSha256: 86D7C89037F741C3AC0F2FA3F1C1DB0BA2E3B910315063625F3F49737011F3CF
- Architecture: arm64
  InstallerUrl: https://statics.teams.cdn.office.net/production-windows-arm64/1.5.00.1870/Teams_windows_arm64.exe
  InstallerSha256: 5C9E21290779FCF86ECABEDC21E57B4576D5C55A6E15AD7900809E35D6D1A754
```